lib/spack/spack/test/cmd/audit.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.main import SpackCommand
+
+
+audit = SpackCommand('audit')
+
+
+def test_audit_packages(mutable_config, mock_packages):
+    """Sanity check ``spack audit packages`` to make sure it works."""
+    audit('packages', fail_on_error=False)
```

Comment:
I'm fine with doing that, I didn't add it in the first place since it requires some boilerplate to setup the test correctly. WWe should probably check that a configuration passing all sanity checks returns an error code of zero.

Reply:
These tests should be moved.

Examples of tests that would be good here:

* Create a package that specifies a "wrong" variant on a dependency and audit it
* Create a "bad" config and audit it

I think we should be testing the audits we provide

Reply:
Tests added in 48d6eaf